### PR TITLE
Iso7763

### PR DESCRIPTION
**Pull request by fragraider**
Added Texas Instruments ISO776x-Q1 series of digital isolators as parts to the "Isolator" library.

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [x] Provide a URL to a datasheet for the symbol(s) you are contributing
-> https://www.ti.com/lit/ds/symlink/iso7763.pdf?ts=1596905577895
- [x] Provide a screenshot of the symbol(s) from the symbol editor with the pin types visible
-> Screenshots at the end of this post.
- [x] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
-> Footprints where available.
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [x] If there are matching footprint PRs, provide link(s) as appropriate
-> NA
- [x] Check the output of the Travis automated check scripts - fix any errors as required
- [x] Give a reason behind any intentional library convention rule violation.
-> S3.1 is violated due to an even number of pins per row wich either spaces some pins far appart or creates the slight offset.

---

Be patient, we maintainers are volunteers with limited time and need to check your contribution against the datasheet. You can speed up the process by providing all the necessary information (see above). And you can speed up the process even more by providing additional info like the screenshot of the symbol editor pin table (or for high pin counts converted to csv) sorted in the same way as the pin table in the datasheet and a direct link to the datasheet page that contains the pin table.

Screenshots Symbols:
billede
billede
billede
billede
billede
billede
billede
billede

Screenshot of Pinout diagrams for all 4 types of chips (Page 4 in datasheet https://www.ti.com/lit/ds/symlink/iso7763.pdf?ts=1596907083909)
billede

### CODE REVIEW

**Comment by cpresser** [comment]
Those look nice. 
I will do a full review soon. 
Upfront, I am not quite sure why you choose the device names as they are. My understanding is that they should be called `ISO776[0-3]?F(DBQ|DW)`. The Q1 suffix is a non functional variant and should be dropped as stated in KLC.
The `F` part should be an alias of the part without it. That leaves 8 variants, each with one additional alias.

**Comment by fragraider** [comment]
Hello cpresser

Thank you for the quick feedback. Still getting very confused with those part numbers. To make thinks worse, I linked the newest data sheet, but was using an older one myself. That one had an extra Q in the name. Lesson learned, should check Manufacturer website directly. Can you let me know if those are right:

ISO7760DBQ alias: ISO7760FDBQ
ISO7760DW alias: ISO7760FDW

ISO7761DBQ alias: ISO7761FDBQ
ISO7761DW alias: ISO7761FDW

ISO7762DBQ alias: ISO7762FDBQ
ISO7762DW alias: ISO7762FDW

ISO7763DBQ alias: ISO7763FDBQ
ISO7763DW alias: ISO7763FDW

**Comment by fragraider** [comment]
Felt confident enough to implement and push those changes. Adjusted the Description regarding high and low output default in case of a non powered "primary" accordingly.

File: Isolator.dcm [hunk 1]
```diff
@@ -696,6 +696,102 @@ K 4Ch Quad Digital Isolator 25Mbps
 F http://www.ti.com/general/docs/lit/getliterature.tsp?genericPartNumber=iso7342fc&fileType=pdf
 $ENDCMP
 #
+$CMP ISO7760DBQ
+D Default high, High-speed, robust EMC, reinforced, six-channel, digital, isolator, SSOP-16
+K 6 channel digital isolator
+F https://www.ti.com/lit/ds/symlink/iso7763.pdf?ts=1596905577895
```
Comment (cpresser) [inline Isolator.dcm:702]: Why did you include the `ts` parameter here?

**Review by cpresser — COMMENTED** [review]
Unsure about the `ts` parameter of the pdf files. Is that documented anywere?
This seems to be the first time that url-parameter is used in the libs.

File: Isolator.dcm (hunk 1 above)
Comment (chschlue) [reply Isolator.dcm:702]: FYI: TI's links to datasheets from component pages include a unix timestamp by default for a couple months or so now.
It's not neccessary and setting the ts back manually doesn't seem to open earlier revisions or anything, so I'd leave it out.

Comment (fragraider) [reply Isolator.dcm]: URLs gibberish to me most of the time. Learned something new and removed the ts parameter.

**Review by cpresser — APPROVED** [review]

**Comment by fragraider** [comment]
Thanks for the tremendous work you guys provide the community with! Very appreciated!

**Comment by cpresser** [comment]
> Thanks for the tremendous work you guys provide the community with! Very appreciated!

You are welcome. This was a good and easy PR. You did draw some wonderful symbols. This was fun for me.
If you want to 'give back', help other users that did submit symbols. Try to answer questions, perhaps do some simple reviews.

**Comment by fragraider** [comment]
I may consider, maybe a few more pushes and i feel confident to actually review someones work. Learned a lot thanks to the patience of you librarians.

**Pull request merged**